src/components/templates/ConversationSpace/ConversationSpace.tsx

```diff
@@ -51,6 +73,12 @@ export const ConversationSpace: React.FunctionComponent = () => {
         </InformationCard>
       </InformationLeftColumn>
       <div className="conversation-space-container">
+        {!seatedAtTable && parentVenueId && parentVenue && (
+          <BackButton
+            onClick={backToParentVenue}
+            locationName={parentVenue.name}
+          />
+        )}
```

Comment:
I'm assuming this is the part that was copied from `JazzBar`?

Does this always want to be going back to the parent venue? Or does it need to have a 'back from table view to main conversation space' button as well?

Reply:
I think Pearl decided that we don't want that because it's not consistent with the jazzbar (and I agree with that)

Reply:
I don't really understand what you're saying here.. I asked if this button is going to show up when I am sitting at at table, because if so, that is something we absolutely do not want to happen.

Reply:
Oh, the button does not show up when you're sitting at the table (because the condition is !seatedAtTable)

Reply:
Awesome, sounds good.

Reply:
The button that goes from the table to the conversation space is in TableHeader